transmart-oauth/grails-app/controllers/org/transmart/oauth/interceptors/OauthInterceptor.groovy

```diff
@@ -1,12 +1,18 @@
-package org.transmartproject.interceptors
+package org.transmart.oauth.interceptors
 
+import org.springframework.beans.factory.annotation.Autowired
+import org.transmartproject.core.audit.AuditLogger
 import org.transmartproject.core.users.User
 
 
 class OauthInterceptor {
     //matches OauthController and all actions of controller by naming convention
-    def auditLogService
+
+    @Autowired(required = false)
+    AuditLogger auditLogService
+    @Autowired
     User currentUserBean
+
     boolean before() {
         auditLogService.report("OAuth authentication", request,
```

Comment:
I might be wrong - but will not cause null pointer exception here if auditLogService fails to get wired (like it is allowed to since it is not required)?